scripts/cnv_wdl/cnv_common_tasks.wdl

```diff
@@ -42,18 +44,22 @@ task CollectCoverage {
     Int? mem
 
     # If no padded target file is input, then do WGS workflow
-    Boolean is_wgs = select_first([padded_targets, ""]) == ""
+    Boolean is_wgs = !defined(padded_targets)
 
     # Sample name is derived from the bam filename
     String base_filename = sub(sub(sub(bam, "gs://", ""), "[/]*.*/", ""), "\\.bam$", "")
-
+ 
+    # Output file name depending on type of coverage
+    String cov_output_name = if (is_wgs && (select_first([transform, ""])  == "RAW")) then "${base_filename}.coverage.tsv.raw_cov" else "${base_filename}.coverage.tsv"
```

Comment:
@sooheelee @knoblett I wasn't aware we had internal scripts that read the gatkDoc-generated json files. The GATK4 json is somewhat different than GATK3, and the python script referenced above appears to be dependent on GATK3. @knoblett Is anyone running (or intending to run) that script on GATK4 json ? If so we probably need to make some changes to get it to work.